Tools/Scripts/libraries/webkitbugspy/webkitbugspy/radar.py

```diff
@@ -546,3 +546,16 @@ def clone(
         if assign:
             result.assign(self.me())
         return result
+
+    def search(self, query):
```

Comment:
I think we want a `FIXME` here noting that we haven't built mocks for this. I think that's appropriate for the time being, but I can see us needing to test this in the future.

Reply:
Good catch, added!